Userland/Libraries/LibWeb/WebDriver/ElementReference.cpp

```diff
@@ -58,20 +58,30 @@ bool represents_a_web_element(JsonValue const& value)
     return value.as_object().has_string(web_element_identifier);
 }
 
-// https://w3c.github.io/webdriver/#dfn-get-a-known-connected-element
+// https://w3c.github.io/webdriver/#dfn-get-a-known-element
 ErrorOr<Web::DOM::Element*, Web::WebDriver::Error> get_known_connected_element(StringView element_id)
 {
     // NOTE: The whole concept of "connected elements" is not implemented yet. See get_or_create_a_web_element_reference().
```

Comment:
This comment is now out of date I think?

Reply:
Not really, we're still papering over the whole concept of connected elements (https://w3c.github.io/webdriver/#dfn-get-or-create-a-node-reference) by just assigning numeric IDs to these nodes.

Reply:
This function just seems to be about elements now, not connected elements. (I can only see one reference to connected elements at all, and it's for ["is stale"](https://w3c.github.io/webdriver/#dfn-is-stale)

Reply:
It's the same concept, they just took "connected" out of the name. So it's now "known element" instead of "known connected element".

See: https://github.com/w3c/webdriver/pull/1705/files#r1049456570

Reply:
Gotcha. Maybe rename `get_known_connected_element()` -> `get_known_element()`?

Reply:
Yeah I was planning on doing a more thorough "catch up to the spec" and "modernize WebDriver" PR after finishing actions :sweat_smile: 

(modernize meaning, this thing should also return NNGCPtr instead of a raw pointer)